pluginClientSdkLib/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
     // plugin so instead the dependency is filtered out from the POM
     // https://code.google.com/p/android/issues/detail?id=68316
     compile group:'com.android.support', name:'support-v4', version:']19,)'
-
+    compile group:'com.android.support', name:'appcompat-v7', version:'[21,)'
```

Comment:
The maven portion of the build script needs to be modified to filter out appcompat-v7 for clients that don't wish to use it.  We're already doing this for support-v4.  Search for "pom.whenConfigured" in this file to find it.